index.html

```diff
@@ -221,6 +221,50 @@ <h2>About me</h2>
             </ul>
         </div>
     </section>
+    <section class="portfolio-form">
+        <div class="form-text">
+            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        </div>
+        <div class="form-area">
+            <form action="https://formspree.io/f/xzboreoy" method="post">
+                <ul class="contact-form">
+                    <li class="name-input">
+                        <input class="names" type="text" name="First Name" maxlength="30" placeholder="First Name">
+                        <input class="names" type="text" name="Last Name" maxlength="30" placeholder="Second Name">
```

Comment:
- [ ] From the template you choose to work on, there is no input for `first name` and` last name`. there is just one input for the `full name`. To make your code DRY and KISS, kindly remove those two inputs (first name and last name).